auslib/db.py

```diff
@@ -20,7 +20,7 @@
 import dictdiffer
 import dictdiffer.merge
 
-from auslib.global_state import cache, dbo
```

Comment:
did you guys delete `dbo` from `auslib.global_state` or is it being used elsewhere apart from here?